VirtoCommerce.CustomerModule.Data/Search/Indexing/MemberDocumentBuilder.cs

```diff
@@ -162,5 +168,34 @@ protected virtual void AddParentOrganizations(IndexDocument document, ICollectio
             document.AddFilterableValues("ParentOrganizations", nonEmptyValues);
             document.AddFilterableValue("HasParentOrganizations", nonEmptyValues?.Any() ?? false);
         }
+
+        protected virtual void IndexDynamicProperty(IndexDocument document, DynamicObjectProperty property)
+        {
+            var propertyName = property.Name?.ToLowerInvariant();
+
+            if (!string.IsNullOrEmpty(propertyName))
+            {
+                var isCollection = property.IsDictionary || property.IsArray;
+                IList<object> values;
+
+                if (property.IsDictionary)
+                {
+                    //add all locales in dictionary to searchIndex
+                    values = property.Values.Select(x => x.Value)
+                                            .Cast<DynamicPropertyDictionaryItem>()
+                                            .Where(x => !string.IsNullOrEmpty(x.Name))
+                                            .Select(x => x.Name)
+                                            .ToList<object>();
+                }
+                else
```

Comment:
If  I had written this code, I would set `else` statement as default for  `values` variable :)